### PR TITLE
Intel(R) oneAPI Collective Communications Library (oneCCL) 2021.13.1

### PR DESCRIPTION
**Pull request by nikitaxgusev**
This PR aligns the contents of the repository with the Intel(R) oneAPI Collective Communications Library 2021.13.1 release.

### CODE REVIEW

File: doc/requirements.txt [hunk 1]
```diff
@@ -29,5 +29,5 @@ sphinxcontrib-htmlhelp==2.0.1
 sphinxcontrib-jsmath==1.0.1
 sphinxcontrib-qthelp==1.0.3
 sphinxcontrib-serializinghtml==1.1.5
-urllib3>=2.2.2
-zipp==3.19.1
```
Comment (shirosankaku) [inline doc/requirements.txt:33]: do i understand it right, that we downgrade the version of components used?

Comment (nikitaxgusev) [reply doc/requirements.txt:33]: @emmwalsh @Maria1Petrova can you please check?

Comment (Maria1Petrova) [reply doc/requirements.txt:33]: shouldn't be the case, perhaps opensource has already newer versions, can we keep new ones?

**Review by bdmoore1 — APPROVED** [review]
The documentation part (.rst files) match what we have on IDZ.

**Review by shirosankaku — APPROVED** [review]

**Review by bdmoore1 — APPROVED** [review]

**Review by Maria1Petrova — APPROVED** [review]

**Pull request merged**